data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature

```diff
@@ -6542,6 +6542,1484 @@ Feature: Data Import integration tests
+    * def uiKey = fileName + randomNumber
+
+    ## Create file definition for FAT-945-1.mrc-file
+    * print 'Before Forwarding : ', 'uiKey : ', uiKey, 'name : ', fileName
+    * def result = call read('common-data-import.feature') {headersUser: '#(headersUser)', headersUserOctetStream: '#(headersUserOctetStream)', uiKey : '#(uiKey)', fileName: '#(fileName)', 'filePathFromSourceRoot' : 'classpath:folijet/data-import/samples/mrc-files/FAT-937.mrc'}
+
+    * def uploadDefinitionId = result.response.fileDefinitions[0].uploadDefinitionId
+    * def fileId = result.response.fileDefinitions[0].id
+    * def importJobExecutionId = result.response.fileDefinitions[0].jobExecutionId
+    * def metaJobExecutionId = result.response.metaJobExecutionId
+    * def createDate = result.response.fileDefinitions[0].createDate
+    * def uploadedDate = result.response.fileDefinitions[0].createDate
+    * def sourcePath = result.response.fileDefinitions[0].sourcePath
+
+    ##Process file
+    Given path '/data-import/uploadDefinitions', uploadDefinitionId, 'processFiles'
+    And param defaultMapping = 'false'
+    And headers headersUser
+    And request
+    """
+    {
+      "uploadDefinition": {
+        "id": "#(uploadDefinitionId)",
+        "metaJobExecutionId": "#(metaJobExecutionId)",
+        "status": "LOADED",
+        "createDate": "#(createDate)",
+        "fileDefinitions": [
+          {
+            "id": "#(fileId)",
+            "sourcePath": "#(sourcePath)",
+            "name": "#(fileName)",
+            "status": "UPLOADED",
+            "jobExecutionId": "#(importJobExecutionId)",
+            "uploadDefinitionId": "#(uploadDefinitionId)",
+            "createDate": "#(createDate)",
+            "uploadedDate": "#(uploadedDate)",
+            "size": 2,
+            "uiKey": "#(uiKey)"
+          }
+        ]
+      },
+      "jobProfileInfo": {
+        "id": "#(jobProfileId)",
+        "name": "FAT-945: Job profile",
+        "dataType": "MARC"
+      }
+    }
+    """
+    When method POST
+    Then status 204
+
+    ## verify job execution for data-import
+    * call pause 180000
+    * call read('classpath:folijet/data-import/features/get-completed-job-execution.feature@getJobWhenJobStatusCompleted') { jobExecutionId: '#(importJobExecutionId)'}
+    * def jobExecution = response
+    And assert jobExecution.status == 'COMMITTED'
+    And assert jobExecution.uiStatus == 'RUNNING_COMPLETE'
+    And assert jobExecution.progress.current == 1
+    And assert jobExecution.progress.total == 1
+    And match jobExecution.runBy == '#present'
+    And match jobExecution.progress == '#present'
+
+    # verify that needed entities created
+    * call pause 10000
+    Given path 'metadata-provider/jobLogEntries', importJobExecutionId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And assert response.entries[0].sourceRecordActionStatus == 'CREATED'
+    And assert response.entries[0].instanceActionStatus == 'CREATED'
```

Comment:
shouldn't the status for Instance and Holdings be UPDATED? and DISCARDED for Item?